src/js/actions/ActionsCreators.js

```diff
@@ -253,8 +253,7 @@ export function setPeriod(payload) {
 
           if (timeLeft < 120 && timeLeft > 0) {
             dispatch(Actions.showNotification({
-              message: text(`This contract can not be traded 
-          in the final 2 minutes before settlement`),
+              message: text('This contract can not be traded in the final 2 minutes before settlement'),
```

Comment:
@qingweibinary yes, because my text editor removes trailing spaces at the end of the line and I wanted to put it in one line in case I forget to add the space back and mess up the translation.